hail/python/hail/context.py

```diff
@@ -47,8 +47,9 @@ def __init__(self, sc=None, app_name="Hail", master=None, local='local[*]',
             assert os.path.exists(hail_jar_path), f'{hail_jar_path} does not exist'
             sys.stderr.write(f'using hail jar at {hail_jar_path}\n')
             conf = SparkConf()
+            conf.set('spark.jars', hail_jar_path)
```

Comment:
the changes to this file make the jar-in-wheel work. It also continues to work locally.